berserk/clients.py

```diff
@@ -1084,6 +1094,30 @@ def decline_takeback(self, game_id):
         """
         return self.handle_takeback_offer(game_id, False)
 
+    def claim_victory(self, game_id):
+        """Claim victory when the opponent has left the game for a while.
+
+        Generally, this should only be called once the `opponentGone` event
+        is received in the board game state stream and the `claimWinInSeconds`
+        time has elapsed.
+
+        :param str game_id: ID of an in-progress game
+        :return: True if successful
+        :rtype: bool
+        """
+        path = f"/api/board/game/{game_id}/claim-victory/"
+        return self._r.post(path)["ok"]
+
+    def berserk_tournament_game(self, game_id):
```

Comment:
maybe `go_berserk` instead? No strong feeling, but currently feels a bit long name. `berserk` alone could be ambiguous given the lib name hence the `go_`

Reply:
Agreed. Updated to `go_berserk`.